test/index.html

```diff
@@ -3,8 +3,6 @@
 <head>
 	<meta charset="utf-8">
 	<title>jQuery Migrate Test Suite</title>
-	<!-- Ensure IE doesn't fall back into oldIE modes -->
-	<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
```

Comment:
Why did you remove this? we still test on IE9-11 so it seems it's better to leave it there.